proposals/0003-cocoapods-support-improvements.md

```diff
@@ -0,0 +1,117 @@
+---
+title: CocoaPods Support Improvements
+author:
+- Orta Therox
+- Eloy Durán
+date: 2018-08-15
+---
+
+# RFC0003: CocoaPods Support Improvements
+
+## Summary
+
+Simplify CocoaPods support in React Native, by reducing the differences between how CocoaPods maps React Native into the existing set of Xcode projects.
+
+## Basic example
+
+CocoaPods support in React Native is a tad tricky, because it’s trying to map a complex set of Xcode projects into a single dependency. The Podspec for React Native does a lot of work under the hood at ~300 LOC. This is, in part, because we use a CocoaPods abstraction called subspecs to consolidate out the React Native library into one single library.
```

Comment:
I would also start from killing those hand-written projects. We have to figure out a way to build them automatically from Buck files which should be (they actually are) source of truth of the build configuration.

Reply:
I read up some of the buck docs - this *might* be possible by having buck output the compiler options into JSON and then having the pod specs capture that, I need to try prototype it to be sure though